components/css/style.css

```diff
@@ -0,0 +1,143 @@
+header {
+  z-index: 99;
+}
+@media (max-width: 1000px) {
+  header img {
+    display: none;
+  }
+  header {
+    padding: 1rem !important;
+  }
+}
+body {
+  overflow: hidden;
+}
+
+section.hero-section a {
+  padding: 0.6rem 1rem;
+}
+
+@media (max-width: 500px) {
+  section.hero-section h1 {
+    font-size: 4rem;
+  }
+}
+@media (max-width: 350px) {
+  section.hero-section h1 {
+    font-size: 2.5rem;
+  }
+  i {
+    font-size: 1rem !important;
+  }
+}
+
+main.container {
+  margin-top: 10vh;
+  display: flex;
+  width: 100%;
+  bottom: 10vh;
+}
+button.menu {
+  display: none;
+  color: var(--light-color);
+}
+.sidebar {
+  margin-top: 2vh;
+  height: 100vh;
+  padding-bottom: 12vh;
+  border-right: 2px solid var(--primary-color);
+}
+.show {
+  display: block !important;
+}
+
+@media (max-width: 1000px) {
+  .sidebar {
+    z-index: 999;
+    display: none;
+    background-color: var(--dark-color-1);
+    backdrop-filter: var(--blur-5);
+    position: absolute;
+    top: 10vh;
+    width: 100%;
+    left: 0;
+  }
+  button.menu {
+    display: block;
+  }
+}
+
+.sidebar-title {
+  padding: 1.2rem;
+  border-bottom: 2px solid var(--primary-color);
+}
+.sidebar ul li {
+  cursor: pointer;
+  padding: 0.5rem;
+  padding-left: 1.5rem;
+  transition: all 0.3s cubic-bezier(0.075, 0.82, 0.165, 1);
+}
+.sidebar ul li:hover {
+  background-color: var(--light-color-8);
+}
+li.active {
+  background-color: var(--light-color-9);
+}
+
+main.component-wrapper {
+  height: 100vh;
+  overflow: hidden;
+  overflow-y: scroll;
+  flex: 1;
+  padding: 5%;
+  padding-left: 8%;
+  padding-bottom: 20vh;
+}
+
+.bg-container {
+  padding: 1rem;
+  display: flex;
+  align-items: center;
+}
+.text-container {
+  margin-bottom: 1rem;
+  display: flex;
+  align-items: center;
+}
+
+/* Code Snippet Syling */
+:root {
+  --radius: 5px;
+  --background: #272822;
+}
+
+pre[class*="language-"] {
+  position: relative;
+  box-sizing: border-box;
+  padding: 1rem;
+  display: flex;
+  min-width: auto;
+  max-width: fit-content;
+  background: var(--dark-color) !important;
+  border-radius: var(--radius);
+  border: 1px solid var(--primary-color);
+  box-shadow: 0 0 1rem var(--primary-color-9);
+}
+code {
+  min-width: auto;
+  max-width: fit-content;
+}
+.code-toolbar {
+  min-width: auto;
+  max-width: fit-content;
+}
+button.copy-to-clipboard-button {
+  background-color: var(--light-color) !important;
```

Comment:
try not to use !important

Reply:
We can't change the property of the predefined class of prismjs.
So I override it with !important